utilities/collectl/buildconfig.yaml

```diff
@@ -15,6 +15,7 @@ spec:
       FROM registry.access.redhat.com/ubi7/ubi:7.8
       RUN yum install https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm -y && \
           yum install collectl-4.3.0-5.el7 pciutils hostname sysvinit-tools -y && \
+          yum remove -y epel-release-7-14.noarch && \
```

Comment:
This will only work if `7-14` is the one installed from the above repo; does `yum remove epel-release-latest-7` work instead to intentionally keep it version agnostic?